packages/fluentui/docs/src/examples/components/Table/Usage/TableExampleNavigable.shorthand.tsx

```diff
@@ -54,7 +56,8 @@ const rowsPlain = [
       moreOptionCell
     ],
     onClick: () => handleRowClick(1),
-    'aria-labelledby': 'name-1 age-1'
+    'aria-labelledby': 'name-1 age-1',
+    children: (Component, props) => <MenuButton menu={contextMenuItems} contextMenu trigger={<Component {...props} />} />
```

Comment:
I think that this will produce a `key` error in console as `key` will be still on a row. The change below will move it to a proper component.

```suggestion
    children: (Component, { key, ...rest }) => <MenuButton menu={contextMenuItems} key={key} contextMenu trigger={<Component {...rest}  />} />
```

Reply:
fixed :)